packages/web-components/src/components/va-checkbox/test/va-checkbox.e2e.ts

```diff
@@ -193,4 +193,17 @@ describe('va-checkbox', () => {
     await inputEl.click();
     expect(await inputEl.getProperty('checked')).toBeTruthy();
   });
+
+  it('fires event for parent when changed', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<va-checkbox aria-describedby="test" label="A label" value="something" />',
+    );
+
+    const changeSpy = await page.spyOnEvent('vaChange');
+    const inputEl = await page.find('va-checkbox');
+    await inputEl.click();
+
+    expect(changeSpy).toHaveReceivedEvent();
+  });
```

Comment:
Do we need this when we already have a very similar test?

https://github.com/department-of-veterans-affairs/component-library/blob/68b523c09e516c6af0ff5c416fbd6d028b57d4af/packages/web-components/src/components/va-checkbox/test/va-checkbox.e2e.ts#L157-L167

Reply:
@bkjohnson good call. I just removed it since it is a duplicate and we can already confirm that the vaChange event has fired via the test you linked and that was what the new test was also trying to confirm